index.php

```diff
@@ -0,0 +1,44 @@
+<?php
+    $db = new PDO('mysql:host=db; dbname=collection', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $query = $db->prepare("SELECT * FROM `playing_cards`;");
```

Comment:
You should replace the * with the specific fields you want to get back, even if its all of them!